CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Change Log
 
+## [3.0.0](https://github.com/auth0/Auth0.swift/tree/3.0.0) (2026-07-15)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)